README.md

```diff
@@ -320,7 +320,7 @@ MethodSpec beyond = MethodSpec.methodBuilder("beyond")
     .addStatement("result.add($T.createNimbus(\"2001\"))", hoverboard)
     .addStatement("result.add($T.createNimbus($T.THUNDERBOLT))", hoverboard, namedBoards)
     .addStatement("$T.sort(result)", Collections.class)
-    .addStatement("return result.isEmpty() $T.emptyList() : result", Collections.class)
+    .addStatement("return result.isEmpty() ? $T.emptyList() : result", Collections.class)
```

Comment:
Now I tend to agree with @mcxtzhang ... if only `Collections.sort` was to be imported, it'd be better to use a single static type import instead of the `*` version.
The additional value is to demonstrate the usage of `.addStaticImport(Collections.class, "*")` with a `class` instance.